design-toolkit/components/src/components/select/select.stories.tsx

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2025 Hypergiant Galactic Systems Inc. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import Placeholder from '@accelint/icons/placeholder';
+import type { Meta, StoryObj } from '@storybook/react';
+import type { ReactNode } from 'react';
+import { Icon } from '../icon';
+import { Options } from '../options';
+import { Select } from './index';
+
+const meta: Meta<typeof Select> = {
+  title: 'Components/Select',
+  component: Select,
+  args: {
+    label: 'Label',
+    description: 'Helper text',
+    errorMessage: '',
+    size: 'medium',
+    isDisabled: false,
+    isInvalid: false,
+    isRequired: true,
+    autoFocus: true,
+    placeholder: 'Select...',
```

Comment:
i see the figma uses foreground/primary-bold for the placeholder, but should it? i think placeholders are generally muted to help distinguish between the placeholder and an actual selected value